frontend/src/utilities/__tests__/valueUnits.spec.ts

```diff
@@ -1,10 +1,89 @@
 import {
+  UnitOption,
+  splitValueUnit,
   isCpuLimitEqual,
   isMemoryLimitEqual,
   isCpuLimitLarger,
   isMemoryLimitLarger,
+  isLarger,
 } from '~/utilities/valueUnits';
 
+describe('splitValueUnit', () => {
+  const options: UnitOption[] = [
+    {
+      name: 'name',
+      unit: 'unit',
+      weight: 1,
+    },
+  ];
+  it('should default back to base if unable to match a legit value', () => {
+    expect(splitValueUnit('', options)).toEqual([1, options[0]]);
+  });
+  it('should return the value and unit', () => {
+    expect(splitValueUnit('1unit', options)).toEqual([1, options[0]]);
+    expect(splitValueUnit('1name', options)).toEqual([1, options[0]]);
+    expect(splitValueUnit('1', options)).toEqual([1, options[0]]);
+    expect(splitValueUnit('1.5unit', options)).toEqual([1.5, options[0]]);
+    expect(splitValueUnit('1.5name', options)).toEqual([1.5, options[0]]);
+    expect(splitValueUnit('1.5', options)).toEqual([1.5, options[0]]);
+  });
+});
+
+describe('isLarger', () => {
+  const unit: UnitOption[] = [
+    {
+      name: 'name',
+      unit: 'unit',
+      weight: 1,
+    },
+  ];
```

Comment:
This requires an additional test with at least 2 different units. Supply another unit type with a different weight for comparison.